src/SW_netCDF.c

```diff
@@ -1751,6 +1797,8 @@ static void fill_domain_netCDF_vals(
 @param[in] primCRSIsGeo Specifies if the current CRS type is geographic
 @param[in] domType Type of domain in which simulations are running
     (gridcell/sites)
+@param[asdf] deflateLevel Level of deflation that will be used for the created
```

Comment:
Could you please fix `@param[asdf] deflateLevel ...` in the documentation to `@param[in] deflateLevel ...`? Here and elsewhere including similar cases such as `@param[adf] deflateLevel ...` and `@param[d] deflateLevel ...`